mLRS/tx-R9M-f103c8/Core/Src/main.cpp

```diff
@@ -56,6 +56,9 @@ int main_main();
 /* USER CODE BEGIN 0 */
 
 /* USER CODE END 0 */
+#ifdef MLRS_USE_BOOTLOADER
+char mlrs_use_bootloader=1; // Indicate to linker script that bootloader will be used
+#endif
```

Comment:
should go to inbetween the` /* USER CODE BEGIN 0 */` and `/* USER CODE END 0 */` enclosures :)

pl also consider the suggestions in the main text :)